climateconnect/climateconnect
# Fix: Removed project page horizontal scroll for tiny screen sizes

## Description
Closes #1057 

Removed the horizontal scroll on project pages that occurred when viewing on a mobile device or small screen by adding a modification to the tab label widths at smaller screen sizes. 

## Test plan

1. Go to any project page on Climate Connect
2. Open Dev tools and go to mobile view. Choose "IPhone SE" or another phone with a narrow screen on the top left
3. Scrolling should no longer be available

## Before landing

1. PR has meaningful title
1. `yarn lint` passes (frontend)
1. `yarn format` passes (frontend)
1. `make format` passes (backend)
